pkg/jx/cmd/opts/common.go

```diff
@@ -259,7 +262,9 @@ func (o *CommonOptions) JXClient() (versioned.Interface, string, error) {
 	if o.factory == nil {
 		return nil, "", errors.New("command factory is not initialized")
 	}
+
 	if o.jxClient == nil {
+		log.Infof("Creating new JX Client")
```

Comment:
Shouldn't we use `Debug` for this? I can get extra verbose if you intent to use it permanently.